README.md

```diff
@@ -1,100 +1,46 @@
-# Template Documentation Site
+# Technical Architecture at the Ministry of Justice
 
-[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=for-the-badge&logo=github&label=MoJ%20Compliant&query=%24.data%5B%3F%28%40.name%20%3D%3D%20%22template-documentation-site%22%29%5D.status&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fgithub_repositories)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/github_repositories#template-documentation-site "Link to report")
+> _Want to give feedback on the documentation? [Open an issue on this repository](https://github.com/ministryofjustice/technical-architecture/issues)._
 
-Use this repository template to create a documentation website using the GOV.UK [Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/) and its [source code](https://github.com/alphagov/tech-docs-template).
+This repository holds the website and documentation for the [Technical Architecture at Ministry of Justice
```

Comment:
I think "*the* Ministry of Justice" is consistent with our usage elsewhere, eg our [Security Guidance](https://security-guidance.service.justice.gov.uk). You've used the definite article at the top of the README, but not in the `service_name` or the index page title.